ironfish/src/rpc/server.ts

```diff
@@ -73,4 +76,21 @@ export class RpcServer {
       this._startPromise = promise
     }
   }
+
+  /** Authenticate the RPC request */
+  authenticate(requestAuthToken: string | undefined): boolean {
+    if (requestAuthToken) {
+      this.logger.debug(`Missing Auth token in RPC request.`)
+      return false
+    }
+
+    const rpcAuthToken = this.node.internal.get('rpcAuthToken')
+
+    if (rpcAuthToken || rpcAuthToken === '') {
+      this.logger.debug(`Missing RPC Auth token in internal.json config.`)
```

Comment:
I think this should be a warning. It means they are trying to use an authenticated adapter without configuring their RpcAuthToken

Reply:
If rpc auth token is empty, it's an internal error because we should have automatically generated the token at beginning.
If user use an authenticated adapter, they should not need to configure if they are running server and client on the same machine. 
If they run them on separate machine, they will receive unauthenticated error. They will need to copy the internal.json to client

Reply:
You mentioned it's an internal error, but you still don't think you want the user to see that error and report the bug as a warning / error?

Reply:
There are different error messages. User will see `Missing authetication` or `Bad authentication`